feature-libs/commerce-quotes/styles/_commerce-quotes-action-links.scss

```diff
@@ -0,0 +1,39 @@
+%cx-commerce-quotes-action-links {
+  section {
+    display: flex;
+    justify-content: flex-end;
+    @include media-breakpoint-down(sm) {
+      padding: 0 1.25rem;
+    }
+    @include media-breakpoint-up(md) {
+      padding-bottom: 1.25rem;
+    }
+    ul {
+      display: flex;
+      list-style: none;
+      margin-bottom: 0;
+      li {
+        padding: 0 1em;
+        text-align: center;
+        border-right: 1px solid var(--cx-color-light);
+        &:last-child {
+          border: none;
+          padding-right: 0;
+        }
+      }
+    }
+  }
+}
+
+html[dir='rtl'] cx-commerce-quotes-action-links {
```

Comment:
Great that you take care of directionality :) Great job!

Reply:
Np, thanks :)